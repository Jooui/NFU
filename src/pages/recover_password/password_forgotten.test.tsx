import React from "react";
import renderer from "react-test-renderer";
import PasswordForgotten from "./password_forgotten";

it("renders correctly", () => {
  const tree = renderer
    .create(<PasswordForgotten></PasswordForgotten>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
