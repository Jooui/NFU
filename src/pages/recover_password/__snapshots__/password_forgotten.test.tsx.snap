// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ion-page"
>
  <ion-content>
    <div
      className="recoverPageContent"
    >
      <div
        className="recoverTitleContainer"
      >
        <img
          alt="icon"
          src="icono.png"
        />
        <span
          className="recoverTitle"
        >
          Recover Password
        </span>
      </div>
      <span
        className="recoverText"
      >
        Enter your email address to receive instructions to reset your password.
      </span>
      <form
        action=""
        className="recoverOptionsContainer_form"
        method="post"
        name="recover_form"
        onSubmit={[Function]}
      >
        <div
          className="recoverInputsContainer"
          id="localOps"
        >
          <div
            className="recoverOption recoverOption--input"
          >
            <ion-input
              onInput={[Function]}
              placeholder="Email"
              type="email"
              value=""
            />
          </div>
          <button
            className="recoverOption recoverOption--submit"
            id="btnrecover"
            type="submit"
          >
            Recover
          </button>
          <div
            className="recoverOption recoverOption--local recoverOption--create"
            onClick={[Function]}
          >
            Back
          </div>
        </div>
      </form>
    </div>
  </ion-content>
</div>
`;
